<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>LoginWindow</class>
    <widget class="QWidget" name="LoginWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>320</width>
                <height>180</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Login</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <!-- Username Field -->
            <item>
                <widget class="QLineEdit" name="username_input">
                    <property name="placeholderText">
                        <string>Enter Username</string>
                    </property>
                </widget>
            </item>
            
            <!-- Password Field -->
            <item>
                <widget class="QLineEdit" name="password_input">
                    <property name="placeholderText">
                        <string>Enter Password</string>
                    </property>
                    <property name="echoMode">
                        <enum>QLineEdit::Password</enum>
                    </property>
                </widget>
            </item>
            
            <!-- Error Label -->
            <item>
                <widget class="QLabel" name="error_label">
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                    <property name="styleSheet">
                        <string>color: red;</string>
                    </property>
                </widget>
            </item>

            <!-- Login Button -->
            <item>
                <widget class="QPushButton" name="login_button">
                    <property name="text">
                        <string>Login</string>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
</ui>
